### PR TITLE
updating workflows - lts

### PR DESCRIPTION
**Pull request by ATorrise**
Action | Description
-- | --									
#6 | [update all workflows to remove deprecation warnings](https://github.com/zowe/zowe-cli-web-help-generator/pull/35/files), [octorelease updates](https://github.com/zowe/zowe-cli/pull/1712)										
#7 | remove Node 14 from workflow matrix

### CODE REVIEW

File: .github/workflows/changelog.yml
Comment (zFernand0) [inline .github/workflows/changelog.yml:1]: Can we update the `awharn/check_changelog_action@v0.0.2` to `v1`?

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -30,10 +30,10 @@ jobs:
 
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@v1
```
Comment (zFernand0) [inline .github/workflows/codeql.yml:36]: please, don't forget to update the `analyze` step too 😋

File: .github/workflows/zowe-cli-plugin.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [14.x, 16.x, 18.x]
```
Comment (zFernand0) [inline .github/workflows/zowe-cli-plugin.yml:15]: we can probably remove the `update-npm` step now that node14 is gone 😋

File: .github/workflows/zowe-cli-plugin.yml [hunk 3]
```diff
@@ -110,7 +110,7 @@ jobs:
     - name: Build Source
       run: npm run build
 
-    - uses: zowe-actions/octorelease@master
```
Comment (zFernand0) [inline .github/workflows/zowe-cli-plugin.yml:113]: I believe the version is just `v1` not `v1s` 😋

**Review by zFernand0 — CHANGES_REQUESTED** [review]

File: .github/workflows/changelog.yml [hunk 4]
```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
     branches:
       - master
-      - zowe-v1-lts
```
Comment (zFernand0) [inline .github/workflows/changelog.yml:7]: some workflows may be missing the `zowe-v?-lts`.
but it's probably fine 😋

**Review by zFernand0 — COMMENTED** [review]
Should we update the version of the CLI in this branch (to remove the Audit failures)?
@t1m0thyj, thoughts?

**Comment by t1m0thyj** [comment]
> Should we update the version of the CLI in this branch (to remove the Audit failures)? @t1m0thyj, thoughts?

Yeah we probably should 🙂 @ATorrise Could you run `npm audit fix` to update the `npm-shrinkwrap.json` and resolve the vulnerability that's causing the Audit check to fail?

**Review by t1m0thyj — CHANGES_REQUESTED** [review]
Will approve once the Audit check is passing

**Review by zFernand0 — CHANGES_REQUESTED** [review]
did you run `npm audit fix --force` ?
if so, it might have applied breaking changes when updating `typedoc`.
nothing wrong, but we may want to address the `typescript` dependency, i.e. update to `"typescript": "^4.9.5"`

**Comment by t1m0thyj** [comment]
If we update `typedoc` through `npm audit fix --force`, it will require us to update a number of other dev deps as well like `typescript`, `ts-jest`, and `jest` so they are all compatible with each other. I'm not opposed to doing this, but don't want to expand the scope of this PR either 😅

@ATorrise It may be easiest to revert back to the last commit before the `npm audit` (https://github.com/zowe/zowe-cli-ims-plugin/pull/57/commits/b894eea020de7835ea099de8377f3b59df5f2c55), and then run `npm audit fix` again without the `--force` flag to update only the Imperative package.

File: .github/workflows/zowe-cli-plugin.yml [hunk 5]
```diff
@@ -40,12 +40,9 @@ jobs:
...
-    - name: Use NPM v8
-      run: npm install -g npm@^8
-
     - name: Install Dependencies
       run: npm ci
-
+      
     - name: Update Dependencies
       id: npm-update
       uses: zowe-actions/octorelease-script@master
```
Comment (t1m0thyj) [inline .github/workflows/zowe-cli-plugin.yml]: Please update to `zowe-actions/octorelease/script@v1`

**Review by t1m0thyj — CHANGES_REQUESTED** [review]

**Review by t1m0thyj — APPROVED** [review]

**Review by zFernand0 — APPROVED** [review]

**Pull request merged**